code42_connector.py

```diff
@@ -359,6 +396,57 @@ def _handle_remove_legal_hold_custodian(self, param, action_result):
         status_message = f"{username} was removed from legal hold matter {matter_id}."
         return action_result.set_status(phantom.APP_SUCCESS, status_message)
 
+    """ FILE EVENT ACTIONS """
+
+    @action_handler_for("hunt_file")
+    def _handle_hunt_file(self, param, action_result):
```

Comment:
There's only one stream file action and it just picks the right method based on detecting the type of hash